arbor/mechcat.cpp

```diff
@@ -114,19 +113,19 @@ struct catalogue_state {
     catalogue_state() = default;
 
     catalogue_state(const catalogue_state& other) {
-        import(other, "");
+        extend(other, "");
```

Comment:
What is the difference between import and extend, since it is used everywhere

Reply:
I renamed the method to avoid clashing with an upcoming in the C++ language (`modules` will bring `import`).
So, now all uses of `catalogue::import` (and `catalogue_state::import`) must be re-written as `extend` instead.

Reply:
got it!